factions/init.lua

```diff
@@ -1,5 +1,17 @@
 factions = {}
+
+--utility function 
+function table.contains(tbl, val)
```

Comment:
thats not what we wanna do. We dont wanna add functions like this. Please write a factions function for this and dont add the function to table #
```suggestion
function table.contains(tbl, val)
=> function factions.contains(tbl, val)
```